### DISCUSSION TITLE
Bug weather

### DISCUSSION THREAD

**Discussion by Dgh1974** [community/community#59886] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi i can not use the data in any way.
Its nit possible to set in var either ?

their are no flows available in homey either for this app ?

**Comment by otter-computer** [comment]
Hi @Dgh1974,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the project's repository. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
